.github/workflows/build-sample-apps.yml

```diff
@@ -0,0 +1,195 @@
+name: Build sample apps
+
+on:
+  pull_request: # build sample apps for every commit pushed to an open pull request (including drafts)
+  push:
+    branches: [ main, feature/* ]
+  release: # build sample apps for every git tag created. These are known as "stable" builds that are suitable for people outside the mobile team.
+    types: [ published ]
```

Comment:
Just a note, `published` has previously not worked for us, especially when kicking off the build release workflow. Its been very flaky and we have to update the CD

Reply:
Removed